src/screens/nocode/NoCode.tsx

```diff
@@ -1,140 +1,34 @@
 import React, {useCallback} from 'react';
-import {ScrollView, StyleSheet, Linking} from 'react-native';
-import {Box, Text, TextMultiline, Toolbar, ButtonSingleLine} from 'components';
+import {ScrollView, StyleSheet} from 'react-native';
+import {Box, Toolbar} from 'components';
 import {useI18n} from 'locale';
 import {useNavigation} from '@react-navigation/native';
 import {useStorage} from 'services/StorageService';
 import {getRegionCase} from 'shared/RegionLogic';
-import {BulletPoint} from 'components/BulletPoint';
-import {BulletPointOrdered} from 'components/BulletPointOrdered';
 import {SafeAreaView} from 'react-native-safe-area-context';
-import {useAccessibilityAutoFocus} from 'shared/useAccessibilityAutoFocus';
-import {captureException} from 'shared/log';
 
-interface ContentProps {
-  title: string;
-  body: string;
-  notCoveredList?: any[];
-  coveredList?: any[];
-  externalLinkText?: string;
-  externalLinkCTA?: string;
-}
-
-const Content = ({title, body, notCoveredList, coveredList, externalLinkText, externalLinkCTA}: ContentProps) => {
-  const autoFocusRef = useAccessibilityAutoFocus(true);
-  const externalLinkButton =
-    externalLinkCTA && externalLinkText ? (
-      <ButtonSingleLine
-        testID="noCodeCTA"
-        variant="bigFlat"
-        text={externalLinkText}
-        onPress={() => Linking.openURL(externalLinkCTA).catch(error => captureException('An error occurred', error))}
-        externalLink
-      />
-    ) : null;
-  return (
-    <Box>
-      <Text
-        testID="noCodeHeader"
-        focusRef={autoFocusRef}
-        variant="bodyTitle"
-        color="bodyText"
-        marginBottom="l"
-        accessibilityRole="header"
-      >
-        {title}
-      </Text>
-      <TextMultiline variant="bodyText" color="bodyText" marginBottom="l" text={body} />
-      {notCoveredList &&
-        notCoveredList.map(item => (
-          <BulletPoint key={item.text} listAccessibile={item.listAccessibile} text={item.text} />
-        ))}
-      {coveredList &&
-        coveredList.map(item => (
-          <BulletPointOrdered
-            key={item.text}
-            orderedListChar={item.number}
-            listAccessibile={item.listAccessibile}
-            text={item.text}
-          />
-        ))}
-      {externalLinkButton}
-    </Box>
-  );
-};
-
-export const NoCodeScreen = () => {
-  const i18n = useI18n();
-  const navigation = useNavigation();
-  const close = useCallback(() => navigation.goBack(), [navigation]);
-  const onChooseRegion = useCallback(() => navigation.navigate('RegionSelect'), [navigation]);
+import {NoRegionView} from './views/NoRegionView';
```

Comment:
These should be at the top of the file

Reply:
They are at the top - maybe it's all the deleted stuff in the diff making it look like they're farther down?